src/platforms/dotnet/guides/aspnet/index.mdx

```diff
@@ -47,8 +57,19 @@ public class MvcApplication : System.Web.HttpApplication
         SentrySdk.CaptureException(exception);
     }
 
+    protected void Application_BeginRequest()
+    {
+        Context.StartSentryTransaction();
+    }
+
+    protected void Application_EndRequest()
+    {
+        Context.FinishSentryTransaction();
```

Comment:
@Tyrrrz we need to ship the .NET SDK before we can merge this.

Reply:
ok, it should be ready